server/server.js

```diff
@@ -7,7 +7,7 @@ var app = express();
 var dbURI = process.env.MONGOLAB_URI || 'mongodb://localhost/rollercost';
 
 // connect to mongo db
-mongoose.connect(dbURI);
```

Comment:
Please don't comment out on org's master. Feel free to send it up then comment it out on your local after. 

This LGTM otherwise; gonna merge.